src/arrayMethodSort.js

```diff
@@ -4,8 +4,18 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => String(a) > String(b),
+  ) {
+    for (let i = 0; i < this.length; i++) {
+      for (let j = i + 1; j < this.length; j++) {
+        if (compareFunction(this[i], this[j]) > 0) {
+          [this[i], this[j]] = [this[j], this[i]];
+        }
+      }
```

Comment:
This is a performance issue. Bubble sort has a worst-case and average complexity of O(n^2), where n is the number of items being sorted. While it's okay to use bubble sort for this task, keep in mind that other sorting algorithms like quicksort or mergesort are generally more efficient for larger lists.